walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,58 @@
 
```

Comment:
nit/option: you probably want to static import commonly used core java standard libraries (as long as it doesn't hurt readability).  (Guava libraries are kind of standard core libraries too). E.g.:

Arrays.asList -> asList
etc.

But maybe don't do:
TimeRange.START_OF_DATE

Reply:
Done.  When you say don't do: TimeRange.START_OF_DATE, do you mean don't static import TimeRange?